website/src/components/TaskSelection/TaskSelection.tsx

```diff
@@ -26,46 +26,57 @@ export const TaskSelection = () => {
           title="Summarize stories"
           link="/create/summarize_story"
         /> */}
+
+        {/* <Tooltip aria-label='A tooltip'label='Ask a question as the user to start the conversation'> */}
+        
         <TaskOption
           alt="Create Initial Prompt"
           img="/images/logos/logo.svg"
           title="Create Initial Prompt"
           link="/create/initial_prompt"
+          label='Prompt the AI to start the conversation'
         />
-        <TaskOption alt="Reply as User" img="/images/logos/logo.svg" title="Reply as User" link="/create/user_reply" />
+        <TaskOption alt="Reply as User" img="/images/logos/logo.svg" title="Reply as User" link="/create/user_reply" label='Keep the conversation going with the user response to the AI' />
         <TaskOption
           alt="Reply as Assistant"
           img="/images/logos/logo.svg"
           title="Reply as Assistant"
           link="/create/assistant_reply"
+          label='Respond to the user as the AI'
+
         />
       </TaskOptions>
-      <TaskOptions key="evaluate" title="Evaluate">
+      <TaskOptions key="evaluate" data-size="Evaluate">
```

Comment:
Accident?

Reply:
Indeed. I saw you said this is currently not being used but I'll leave it up to you if it should be committed for possible future cases if we go back to these components or if they will be deleted all together.